community_images/traefik/ironbank/image.yml

```diff
@@ -25,6 +25,9 @@ repo_sets:
   - opensource/traefik/traefik:
```

Comment:
I think we are not removing the old tags from image.yml, @Kirandeep-Singh-Khehra can you please take a look?

Reply:
Initially the idea was to append/concatenate new tags to older one. But the new changes pushed will(are expected to) do the following:
1. Bitnami `image.yml` s will have tags that are present in bitnami_tags.yml (only and no old or other than)
2. Ironbank `image.yml`s will have only one tag that is the latest(excluding string "latest") tag in the list of tags mentioned in `hardening_manifest.yml` on repo1. Url for hardening manifest is taken from the `source_image_readme` key in image.yml. So, presence of different tags/`hardening_manifest.yml` in different repo is not considered. As current data in `image.yml` is not enough to track those repos.